### DISCUSSION TITLE
How to monitor power consumption?

### DISCUSSION THREAD

**Discussion by Ocelotinho** [microchip-pic-avr-solutions/avr-iot-cellular-arduino-library#26] [Q&A] [answered]
Hello!

How can I monitor the power consumption of my AVR Cellular Mini ? 
I've seen a screenshot at the bottom of the GPS tracker example [page](https://iot.microchip.com/docs/arduino/examples/GPS%20Tracker/Arduino%20Sketch). 

What software was used?

![gps_power 31ddbe4 1030](https://github.com/user-attachments/assets/5470cf1b-9b23-4e0e-825b-fcb10d4a68d4)

Thank you

**Comment by simengangstad** [comment, marked as answer]
Hello, that screenshot is from [Atmel Data Visualizer](https://www.microchip.com/en-us/tools-resources/debug/atmel-data-visualizer), using a [power debugger](https://www.microchip.com/en-us/development-tool/atpowerdebugger).
